code/modules/surgery/organs/pain.dm

```diff
@@ -1,62 +1,47 @@
-mob/var/list/pain_stored = list()
-mob/var/last_pain_message = ""
-mob/var/next_pain_time = 0
+/mob/living/carbon/human
+	var/last_pain_message = ""
+	var/next_pain_time = 0
 
 // partname is the name of a body part
 // amount is a num from 1 to 100
-mob/living/carbon/proc/pain(var/partname, var/amount, var/force, var/burning = 0)
-	if(stat >= 2) return
+/mob/living/carbon/human/proc/pain(partname, amount)
+	if(stat >= UNCONSCIOUS)
+		return
 	if(reagents.has_reagent("sal_acid"))
 		return
 	if(reagents.has_reagent("morphine"))
 		return
 	if(reagents.has_reagent("hydrocodone"))
 		return
-	if(world.time < next_pain_time && !force)
+	if(world.time < next_pain_time)
 		return
-	if(amount > 10 && istype(src,/mob/living/carbon/human))
-		if(paralysis)
-			AdjustParalysis(-round(amount/10))
```

Comment:
This is probably the reason for a few weird random wake-ups during edge cases in surgery.